druid/src/widget/textbox.rs

```diff
@@ -31,6 +31,8 @@ const TEXT_INSETS: Insets = Insets::new(4.0, 2.0, 0.0, 2.0);
 
 const CURSOR_BLINK_DURATION: Duration = Duration::from_millis(500);
 
+const MAC_OR_LINUX: bool = cfg!(any(mac, windows));
```

Comment:
Should this be `cfg!(any(mac, linux));`?

Reply:
um.

yes, definitely.